test/data/build.get.yaml

```diff
@@ -10,5 +10,14 @@ stats:
   blockedStartTime: '2017-01-06T01:49:50.384359267Z'
   queueEnterTime: '2017-01-06T01:49:50.384359267Z'
   imagePullStartTime: '2017-01-06T02:49:50.384359267Z'
+parentBuilds:
+  111:
+    eventId: 1,
```

Comment:
what if there's a job named `eventId`?

Reply:
That’s a really weird job name but totally possible. Maybe we can put all the jobs under keyword “jobs”. Then we can access by “111.jobs.A”

Reply:
will need to change API logic (https://github.com/screwdriver-cd/screwdriver/pull/1805) to check build status based on `111.jobs.A` instead of `111.A`